Library/src/Systems/Quota/Quota.cpp

```diff
@@ -367,4 +422,67 @@ const TierFeatures StringToTierFeatureEnum(const csp::common::String& Value)
 	return TierFeatures::SpaceOwner;
 }
 
+XErrorCodeTypes StringToXErrorCodeTypeEnum(const csp::common::String& Value)
+{
+	if (Value == "group_spaceownerquota")
```

Comment:
The string case here does not match what is listed in the CHS TDD, is what you have here correct? Eg: **Group_SpaceOwnerQuota**

Reply:
Same for the ones below

Reply:
The TDD seems to be wrong